server/actions/Contentful.ts

```diff
@@ -207,4 +208,50 @@ export async function getJournalEntryByType(type: string){
         if(error) console.error(error);
         return {};
     }
-}
\ No newline at end of file
+}
+
+/**
+* @param id ID of the JournalEntry to be updated
+* @returns True if successful. False if unsuccessful.
+*/
+export async function updateJournalEntryReviewStatus(id: string){
+    try{
+        const space = await client.getSpace(process.env.CONTENTFUL_SPACE as string);
+        const environment = await space.getEnvironment(process.env.CONTENTFUL_ENVIRONMENT as string);
+        const entry = await environment.getEntry(id);
+        //Review status would only ever go from false to true, so there's no need to specify what the status would be.
+        entry.fields.reviewed['en-US'] = true;
+        await entry.update();
+        return true;
+
+
+    } catch (error){
+        if(error) console.error(error);
+        return false;
+    }
+}
+/**
+* @param id The unique journal identifier given by contentful.
+* @returns True if successful or false if there's an error.
+*/
+export async function deleteJournalEntryById(id: string){
```

Comment:
I think deleting on rejecting is probably best to save space on Contentful, so this part looks good

Reply:
All the credit goes to Kemal, he did a majority of the backend work for journal entries. Wouldn't have been possible without his help